update the BQ permission requirement section (#5098)
segmentio/segment-docs: src/connections/reverse-etl/reverse-etl-source-setup-guides/bigquery-setup.md

> `bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists

Reviewer: ```suggestion
`bigquery.datasets.get` | This allows Segment to determine if the aforementioned dataset exists.
```

> `bigquery.jobs.create` | This allows Segment to execute queries on any datasets/tables your model query references and manage tables that Segment uses for tracking

Reviewer: ```suggestion
`bigquery.jobs.create` | This allows Segment to execute queries on any datasets/tables your model query references and manage tables that Segment uses for tracking.
```

> If you don't wish to grant `bigquery.datasets.create` access, you may create this dataset yourself, and Segment needs the following permissions:

Reviewer: ```suggestion
If you don't want to grant Segment `bigquery.datasets.create` access, you can create your own `__segment_reverse_etl` dataset and give Segment the following permissions: 
```

Permission | Details
---------- | --------

> `bigquery.datasets.get` | This allows Segment to determine if the `__segment_reverse_etl` dataset exists
> `bigquery.tables.create` | This allows Segment to determine if the tables Segment uses to track state in `__segment_reverse_etl` dataset exists
> `bigquery.jobs.create` | This allows Segment to execute queries on any datasets/tables your model query references and manage tables that Segment uses for tracking

Reviewer: ```suggestion
`bigquery.datasets.get` | This allows Segment to determine if the `__segment_reverse_etl` dataset exists.
`bigquery.tables.create` | This allows Segment to determine if the tables Segment uses to track state in the `__segment_reverse_etl` dataset exists.
`bigquery.jobs.create` | This allows Segment to execute queries on any datasets or tables your model query references, and also allows Segment to manage tables used for tracking.
```

Author: Hey @stayseesong, I assume the above suggested change for line 47 applies to line 40 as well?

> The `bigquery.datasets.*` permissions can be scoped only to the `__segment_reverse_etl` dataset. 

Reviewer: @reneewang21 just want to make sure, should there be a `*` for bigquery.datasets.

Author: yes, there are two `bigquery.datasets` permissions we mentioned, one is for `create` and the other is for `get`.
